util/test/test_query.py

```diff
@@ -158,6 +106,15 @@ def test_all(tmp_path):
     sampling_accounts_addrs = [a.address.lower() for a in sampling_test_accounts]
     delegation_accounts = [a.address.lower() for a in accounts[:2]]
     delegation_accounts.append(zerobal_delegation_test_acc.address.lower())
+
+    # test ghost consume
+    (V0, _, _) = query.queryVolsOwnersSymbols(rng, CHAINID)
+    assert V0[CO2_addr][ghost_consume_nft_addr] == approx(1.0, 0.5)
+    (V0, _, _) = query._queryVolsOwners(ST, FIN, CHAINID)
+    assert V0[CO2_addr][ghost_consume_nft_addr] == 21.0
+    swaps = query._querySwaps(ST, FIN, CHAINID)
+    assert swaps[CO2_addr][ghost_consume_nft_addr] == approx(1.0, 0.5)
```

Comment:
First - all the other tests are called via sub-functions. Eg the following:
```python
    _test_getSymbols()
    _test_queryVolsOwners(CO2_addr, ST, FIN)
   #(and 10 more)
```

Therefore please move this code-block into its own sub-function named e.g. `_test_ghostConsume()`. Pass in args as needed.

Second - it's always best to have simple& happy-path tests first, like `_test_getSymbols()`. Ghost consume is more advanced and non-happy-path. Therefore please move `_test_ghostConsume()` to after the simple tests. E.g. after these tests:
```python
    # end-to-end tests
    _test_end_to_end_without_csvs(CO2_sym, rng)
    _test_end_to_end_with_csvs(CO2_sym, rng, tmp_path)

   #(put ghost consume test here)
```